subsys/bluetooth/shell/iso.c

```diff
@@ -417,6 +417,8 @@ static int cmd_big_create(const struct shell *sh, size_t argc, char *argv[])
 	param.bis_channels = bis_channels;
 	param.num_bis = BIS_ISO_CHAN_COUNT;
 	param.encryption = false;
+	param.packing = BT_ISO_PACKING_SEQUENTIAL;
```

Comment:
In order to avoid the same problem when you add something else, maybe memset or initialize the entire struct to zero?
Would _INIT macros be helpful?

Reply:
I did consider the memset approach. I can add that if you like

Didn't consider the _INIT macro approach, but definitely something that could also make sense. 

Please let me know if you want either approach, or if this is good enough as is.

Reply:
This was just a suggestion. I'm fine with leaving it as is.

Reply:
I'll try to keep that in mind, and possibly add that later :)